protocol/git/README.md

```diff
@@ -1,120 +1,50 @@
-Git Protocol
-============
-
-A guide for programming within version control.
-
-Maintain a Repo
----------------
-
-* Avoid including files in source control that are specific to your
-  development machine or process.
-* Delete local and remote feature branches after merging.
-* Perform work in a feature branch.
-* Rebase frequently to incorporate upstream changes.
-* Use a [pull request] for code reviews.
-
-[pull request]: https://help.github.com/articles/using-pull-requests/
-
-Write a Feature
----------------
-
-Create a local feature branch based off master.
-
-    git checkout master
-    git pull
-    git checkout -b <branch-name>
-
-Rebase frequently to incorporate upstream changes.
-
-    git fetch origin
-    git rebase origin/master
-
-Resolve conflicts. When feature is complete and tests pass, stage the changes.
-
-    git add --all
-
-When you've staged the changes, commit them.
-
-    git status
-    git commit --verbose
-
-Write a [good commit message]. Example format:
```

Comment:
Yes, we should select the items in the original doc that we want to keep and integrate them into the new doc. Definitely agree that guidelines on commit messages would be useful to keep.